atlp-rwanda/e-commerce-ones-and-zeroes-bn
# [Delivers #187355107] ft disable account

### What does this PR do?

This feature allows admins to disable and user accounts

How should this be manually tested?

This feature can be tested via "http://localhost:3000/docs/" under the "/users/disable/${userId} route

In order to login, you should have signed up and verify your account.

### Step1: Login

![20240521_20h31m59s_grim](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/42178217/850d703a-503f-40f8-b8cf-51b040a27053)

![20240521_20h34m03s_grim](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/42178217/13a89210-96ae-4be1-9c08-b0cae85bb940)


### Step2: Add token to Authorization Header

![20240521_20h35m29s_grim](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/42178217/343d8d18-189c-4e6f-891d-036eef14352e)

### Step3: Execute the /api/users/disable/{id} route with reason in the body

![20240521_20h37m46s_grim](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/42178217/a485e53d-8579-41ca-8426-793d53b193c5)

![20240521_20h39m26s_grim](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/42178217/ddd75af0-6ccc-4b15-a0af-e4ddb6c14df5)

What are the relevant pivotal tracker stories?

Pivotal Tracker Id: #187355107
